sk0-front/src/app/global-services/notification.service.ts

```diff
@@ -16,4 +19,21 @@ export class NotificationService {
     });
     await toast.present();
   }
+
+  async alertSuccess(message: string) {
```

Comment:
do we need both similar methods or is it worth to create one method call displayAlert() and we can pass an object with union type like "notification" | "error" | "warning" ? the notification without a header just the message, the error  with an error header, the message and an OK button, and lastly warning with the attention header, message, and two buttons bellow for cancelling and OK

Reply:
Yeah, we can do that, but we can wait for the ngrx actions, and we can refactor there too,  I don't mind.